### PR TITLE
[Feature] ADD OP_CREATE_DB_V2 to support create database with json serialized

### PR DESCRIPTION
**Pull request by HangyuanLiu**
## What type of PR is this:
- [ ] BugFix
- [x] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes:

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 14 / 26 (53.85%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/persist/EditLog.java|1|5|20.00%|[171, 172, 173, 174]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|2|8|25.00%|[429, 431, 432, 433, 435, 437]|
|:large_blue_circle:|com/starrocks/persist/CreateDbInfo.java|8|10|80.00%|[39, 43]|
|:large_blue_circle:|com/starrocks/journal/JournalEntity.java|3|3|100.00%|[]|

**Review by imay — DISMISSED** [review]

File: fe/fe-core/src/main/java/com/starrocks/persist/EditLog.java [hunk 1]
```diff
@@ -1086,8 +1092,8 @@ public void logSaveDeleteAutoIncrementId(AutoIncrementInfo info) {
         logEdit(OperationType.OP_DELETE_AUTO_INCREMENT_ID, info);
     }
 
-    public void logCreateDb(Database db) {
-        logEdit(OperationType.OP_CREATE_DB, db);
+    public void logCreateDb(CreateDbInfo createDbInfo) {
+        logEdit(OperationType.OP_CREATE_DB_V2, createDbInfo);
```
Comment (gengjun-git) [inline fe/fe-core/src/main/java/com/starrocks/persist/EditLog.java:1096]: The write cannot be changed now.

Comment (HangyuanLiu) [reply fe/fe-core/src/main/java/com/starrocks/persist/EditLog.java:1100]: done

**Review by gengjun-git — APPROVED** [review]

File: fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java [hunk 2]
```diff
@@ -423,6 +431,17 @@ public void replayCreateDb(Database db) {
         }
     }
 
+    public void replayCreateDb(CreateDbInfo createDbInfo) {
+        tryLock(true);
```
Comment (nshangyiming) [inline fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java:435]: should use lock ? we cannot timeout here i think

**Review by nshangyiming — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 0 / 25 (00.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/persist/CreateDbInfo.java|0|10|00.00%|[33, 34, 35, 36, 39, 43, 48, 49, 52, 53]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|0|6|00.00%|[435, 437, 438, 439, 441, 443]|
|:large_blue_circle:|com/starrocks/persist/EditLog.java|0|6|00.00%|[171, 172, 173, 174, 1100, 1101]|
|:large_blue_circle:|com/starrocks/journal/JournalEntity.java|0|3|00.00%|[214, 215, 216]|

**Pull request merged**